scss/core/base.scss

```diff
@@ -356,22 +356,25 @@ table.mce-item-table {
   box-shadow: none;
```

Comment:
If the class is used on something that has a custom box-shadow, then box-shadow shouldn't be removed.

```suggestion
```

Reply:
Fixed